src/tests/OutputPresenter.test.ts

```diff
@@ -0,0 +1,94 @@
+import { OutputPresenter } from "../OutputPresenter";
+import { templates } from "../Template";
+describe("output presenter", () => {
+  let presenter: OutputPresenter;
+  let mood = templates[0];
+
+  window.HTMLMediaElement.prototype.play = jest.fn();
+  window.HTMLMediaElement.prototype.pause = jest.fn();
+  window.HTMLMediaElement.prototype.load = jest.fn();
+  beforeEach(() => {
+    presenter = new OutputPresenter();
+  });
+
+  describe("playVideo", () => {
+    it("should set play viriable as true when played second is greater than total duration", () => {
+      presenter.overallPlayedSeconds = 10;
+      presenter.currLength.totalDuration = 5;
+      presenter.music = new Audio(mood.tracks[0].musicTrack);
+      presenter.playVideo();
+      expect(presenter.music.play).toHaveBeenCalled();
+      expect(presenter.play).toBe(true);
+    });
+
+    it("should set play as true", () => {
+      presenter.playVideo();
+      expect(presenter.play).toBe(true);
+    });
+  });
+
+  describe("pauseVideo", () => {
+    it("should set play as false", () => {
+      presenter.pauseVideo();
+      expect(presenter.play).toBe(false);
+    });
+  });
+
+  describe("setPlayingMedia", () => {
+    it("should set playingMedia to the passed in value", () => {
+      presenter.setPlayingMedia(0);
+      expect(presenter.playingMedia).toEqual(0);
+    });
+  });
+
+  describe("incrementPlayingMedia", () => {
+    it("should increase the value of playingMedia by 1", () => {
+      expect(presenter.playingMedia).toEqual(0);
+      presenter.incrementPlayingMedia();
+      expect(presenter.playingMedia).toEqual(1);
+    });
+  });
+
+  describe("adjustSound", () => {
+    it("should decrease the volume by 0.01 given currentTime is greater than (endofmusic - VOLUMEPOINT)", () => {
```

Comment:
try to make your test names very clear, without referencing values (otherwise if you change the value, you need to change the test name)
eg. `should decrease the volume when the music is near the end`